platform/error/platform-error-impl/package.json

```diff
@@ -44,7 +38,7 @@
     "node-fs": "0.1.7",
     "node-options": "0.0.3",
     "path": "0.4.9",
-    "casperjs": "~1.1.0",
+    "casperjs": "^1.1.0-beta3",
```

Comment:
Do not use the version modifiers like ^ or ~ since they make our build/release indeterministic.